### PR TITLE
Conditionalize specimen verification in tests

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
We want study tests to run both with and without the specimen module present. This PR conditionalizes specimen setup and verification steps based on whether the module is currently deployed.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/2005

### CODE REVIEW

File: src/org/labkey/test/util/StudyHelper.java [hunk 1]
```diff
@@ -48,6 +49,7 @@
     public static final File SPECIMEN_ARCHIVE_B = TestFileUtils.getSampleData("study/specimens/sample_b.specimens");
 
     protected BaseWebDriverTest _test;
+    private Boolean _specimenModulePresent = null;
```
Comment (labkey-tchad) [inline src/org/labkey/test/util/StudyHelper.java]: Make this static?

**Review by labkey-tchad — APPROVED** [review]

File: src/org/labkey/test/tests/StudyBaseTest.java [hunk 2]
```diff
@@ -212,6 +214,11 @@ protected void exportStudy(boolean zipFile, boolean exportPhi, PhiSelectType exp
         waitForText("Visit Map", "Cohort Settings", "QC State Settings", "CRF Datasets", "Assay Datasets",
                 "Dataset Data", "Specimens", "Specimen Settings", "Participant Comment Settings");
```
Comment (labkey-tchad) [inline src/org/labkey/test/tests/StudyBaseTest.java]: Remove "Specimens" and "Specimen Settings" from the list of text to wait for.

**Review by labkey-tchad — CHANGES_REQUESTED** [review]

File: src/org/labkey/test/util/StudyHelper.java (hunk 1 above)
Comment (labkey-adam) [reply src/org/labkey/test/util/StudyHelper.java]: Sure

File: src/org/labkey/test/tests/StudyBaseTest.java (hunk 2 above)
Comment (labkey-adam) [reply src/org/labkey/test/tests/StudyBaseTest.java]: Oops

**Review by labkey-tchad — APPROVED** [review]

**Pull request merged**